_apprenticeship/interview-process.md

```diff
@@ -0,0 +1,36 @@
+---
+position: 42
+layout: features3col
+title: The Interview Process
+subtitle: In the process we want to learn if our expectations match, yours and ours.
+          We want to learn about your motivation and align on our expectations for the apprenticeship.
+          During the interview process we want to learn what you want to get our of the apprenticeship
+          what we can provide by mentoring and helping you grow.
+          Every step helps us to decide if we want to continue the process.
+features:
+  - title: Application Review
+    icon: envelope-letter
+    text: After applying we will review your CV and cover letter. As mentioned above a cover letter is required, 
+          we want to learn about why you think the apprenticeship is right for you.
+          Depending on the results of this first review we will decide if we want to proceed with the application 
+          process.
+  - title: Online Interview
+    icon: earphones
+    text: The first interview will be 1.5h where we introduce the company and want to get to know each other.
+          We want to learn about motivations and your tech background, you ambitions and passion that led you to 
+          apply for the apprenticeship.
+  - title: The Challenge
+    icon: home
+    text: The second part will be a challenge that we send out, which you solve on your own and 
+          we review the code and will get back with a couple specific inputs, so you can improve
+          and advance the code.
+  - title: On-Site Interviews
+    icon: bubbles
+    text: In this step there will be a set of interviews in our offices. It will be one pair-programming,
+          one with our People.Team and two more that will be with people outside of tech, so we get the
```

Comment:
Let's write `"People.Team"` or `People-Team`. Otherwise it looks like its the end of the sentence

Reply:
👍